stable/pomerium/templates/ingress.yaml

```diff
@@ -0,0 +1,36 @@
+{{- if .Values.ingress.enabled -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ include "pomerium.fullname" }}
```

Comment:
@desimone you are missing the `.` after the template name

Reply:
@victornoel 

> @desimone for the record here are some changes I made to this PR so that I can run pomerium with self-signed certificates and using only internal urls: [victornoel@6fcd78a](https://github.com/victornoel/helm-charts/commit/6fcd78ad2d8cb3926c5e5513a92a9450625745bf)

Very helpful, thank you. 

> I generated my own config file and ingresses for this.
> 
> Normally most of the changes corresponds to comment I made here.
> 
> One thing is missing from there: I had to add the env variable `OVERRIDE_CERTIFICATE_NAME` set to `example.com` to make it work with the self-signed certificates.

I've been struggling with how to best approach creating a default ingress template.  Are you generating a wildcard self-signed certificate for a custom domain then routing based on host similar to [this](https://www.pomerium.io/docs/examples.html#ingress-yml) ?

Thanks again for the review and feedback!